db-ops/create-db-user.sh

```diff
@@ -0,0 +1,29 @@
+# docker image with postgres client only
+DOCKER_IMAGE_TAG=governmentpaas/psql:latest
+
+DB_ENDPOINT=$(aws rds describe-db-instances --region=$REGION --query "DBInstances[?DBName=='$PROJECT_NAME'].Endpoint.Address" | jq '.[0]' |  sed "s/\"//g")
```

Comment:
Another alternative we may want to explore here which would make this easier to use is in the aws-eks repo we could create an "externalname" service in the cluster which points to the database, then we could refer to it by a static name rather than having to get the RDS hostname.

Also if you use `jq -r` you don't need that `sed` at the end.